homework/index.js

```diff
@@ -0,0 +1,21 @@
+const repoSelectElement = document.getElementById('repositories');
+const unContainer = document.querySelector('.right__div-username');
+const reposContainer = document.querySelector('.right__div-repos');
+const avatarContainer = document.querySelector('.right__div-avatar');
+
+
+const CONTRIBUTORS_URL = async () => {
+  const API_CALL = await fetch(
+    'https://api.github.com/repos/HackYourFuture/tdd-game/contributors',
+  );
+  const data = await API_CALL.json();
+  return { data };
```

Comment:
You're 'wrapping' this data (the array) in a new object (using the ES6 'object property value shorthand' notation). So instead of directly returning the array, you're returning an object with a property 'data' which contains the array. That's not really necessary - just return the array.